src/compiler/scala/tools/nsc/typechecker/ContextErrors.scala

```diff
@@ -622,8 +622,10 @@ trait ContextErrors {
         NormalTypeError(tree, fun.tpe+" does not take parameters")
 
       // Dynamic
-      def DynamicVarArgUnsupported(tree: Tree, name: Name) =
-        issueNormalTypeError(tree, name+ " does not support passing a vararg parameter")
+      def DynamicVarArgUnsupported(tree: Tree, name: Name) = {
+        issueNormalTypeError(tree, name + " does not support passing a vararg parameter")
+        setError(tree)
```

Comment:
The new errors (reported in the test case) are not helpful / confusing, I'd say let's remove this again.

Reply:
I just found it rather irregular/strange that the "varargs not supported" error masked this (IMO bigger and more important) one.

Reply:
OK, you convinced me :)